CODING_STANDARDS.md

```diff
@@ -244,12 +244,54 @@ activateRipple() {
 }
 ```
 
+##### Selectors
+* Component selectors should be lowercase and delimited by hyphens.
+* Directive selectors should be camel cased.
+
 #### Inheritance
 
 Avoid using inheritance to apply reusable behaviors to multiple components. This limits how many
 behaviors can be composed. Instead, [TypeScript mixins][ts-mixins] can be used to compose multiple
 common behaviors into a single component.
 
+#### MDC checks
+To ensure backwards compatability, we check that tests written for MDC Components include all of
+the same tests that the non-MDC version had. Similarly, we check that the public API of MDC
+Components match that of the non-MDC version.
+
+In the case where old tests no longer make sense and should be omitted or the public API should be
+changed, you can do so in [scripts/check-mdc-tests-config.ts](https://github.com/angular/components/blob/master/scripts/check-mdc-tests-config.ts) and
+[scripts/check-mdc-exports-config.ts](https://github.com/angular/components/blob/master/scripts/check-mdc-exports-config.ts).
+Remember to leave a comment explaining why the change was necessary.
+
+#### Coercion
+When defining public properties, use our coercion API to coerce `@Input`s into specific types.
```

Comment:
I think that this is covered by the `Boolean arguments` section already.

Reply:
From what I can tell, the `Boolean arguments` section only talks about how we should avoiding adding boolean arguments to a method in cases where the argument means "do something extra". Can you provide a link to where coercion is already documented?

Reply:
You're right, I didn't read it through. One note on your change is it makes it sound like we should be coercing all public properties, but it's only relevant for numbers and booleans.